### DISCUSSION TITLE
Appreciation thread / Tråden för inbördes beundran

### DISCUSSION THREAD

**Discussion by ispjstfu** [jonkpirateboy/Pass-fur-alle#101] [General]
Inget issue. Vill bara säga tack! Tycker vi saknar en tråd för inbördes beundran där vi tackar både jonkpirateboy och alla andra som fixar eventuella problem som uppstår.

Jag fixade precis en tid åt en äldre granne som knappt löser det tekniska överhuvudtaget, efter att ha skrattat en stund åt uttalandet "är det inte bara att gå ned och ta en nummerlapp? Så brukade det vara, men det är nog många år sedan nu..." (Vi bor i huset mittemot passpolisen här, men får ingen förtur för det. Dålig grannsamverkan...)

Så, ja... TACK! Ni är bäst!

**Comment by cquencerr** [comment]
+1 på den!!

**Comment by Falkens-swe** [comment]
Kan inte annat än hålla med... hjälpte en kompis inför sommaren då dotterns pass hade gått ut och fick denna tillbaka igår.
"_Fan va sjukt att du lyckades lösa en passtid i [Staden] med så kort varsel. Jag har legat spikrak och bara stirrat i taket i chock sen dess._"

**Comment by linussvahn** [comment]
+100 på den.

Vi trodde vi var safe i februari när dessa problem började florera, men insåg att sones pass bara var giltigt i tre år (och inte fem som våra) så fick boka en passtid direkt, men då var första lediga tid 23:e juni i Stockholm.. 

Efter drygt en timme med PfA så har jag nu en tid redan imorgon, semestern räddad! 

Tack Johan och alla inblandade! 🥳🥳🥳

**Comment by CrazySwede78** [comment]
Instämmer helhjärtat! Fick en passtid i NOVEMBER i år fär mig och sambon. både I Stockholm och Uppsala. Efter en dag med detta script så är det 5 dagars väntetid i stället för 7 månader. Verkligen, jättetack!

**Comment by xRadne** [comment]
Om jag bara hade använt detta script för en månad sedan hade jag sluppit lägga 10-20 minuter om dagen att på arbetstid sitta och uppdatera i hopp om att det ska dyka upp återbudstider.

Detta var fantastiskt smidigt! 😎✨

**Comment by metinar** [comment]
stort tack!

**Comment by ispjstfu** [comment]
Så kul att det fortfarande funkar. Fixade tid för ID-kort åt en vän igår.

... Nu måste vi bara klura ut hur man hackar Q-Matic! Sjuka köer för att hämta ut pass/ID på vissa platser. Vore ju magiskt att alltid få nummer 1 i kön hos alla myndigheter, i delikatessdisken och hos postombudet 😂

**Comment by jonkpirateboy** [comment]
Flyttade denna hit. :)
